pkg/provider/statefulsets_test.go

```diff
@@ -92,3 +95,35 @@ func TestIsStatefulSetReady(t *testing.T) {
 		assert.Equal(t, tc.expectedOutput, testSS.IsStatefulSetReady())
 	}
 }
+
+func TestGetUpdatedStatefulset(t *testing.T) {
+	testCases := []struct {
+		testNamespace string
+	}{
+		{ // Test Case #1 - Test with valid namespace 'testNS1'
+			testNamespace: "testNS1",
+		},
+		{ // Test Case #2 - Test with valid namespace 'testNS2'
+			testNamespace: "testNS2",
+		},
```

Comment:
A test case for the error output of `GetUpdatedStatefulset` would also be nice to have.

Reply:
Good call, I'll add one.